test/graphinfo.jl

```diff
@@ -57,6 +60,13 @@ single_parent_m = Model(μ = (1.0, () -> 3, :Logical), y = (1.0, (μ) -> MvNorma
 @test typeof(single_parent_m) == Model{(:μ, :y)}
 @test typeof(single_parent_m.g) == GraphInfo{(:μ, :y)}
 
+# test setindex
+
+@test_throws AssertionError setvalue!(m, @varname(s2), 0.0)
+@test_throws AssertionError setvalue!(m, @varname(s2), (1.0))
```

Comment:
```suggestion
@test_throws AssertionError setvalue!(m, @varname(s2), (1.0,))
```
or
```suggestion
@test_throws AssertionError setvalue!(m, @varname(s2), 1.0)
```
which did you mean?

Reply:
The first. The comma hits me again...